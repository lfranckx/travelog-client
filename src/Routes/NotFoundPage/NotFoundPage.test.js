import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import NotFoundPage from './NotFoundPage';

describe(`NotFoundPage component`, () => {
    it(`renders without exploding`, () => {
        expect(
            shallow(
                <NotFoundPage />
            ).length
        ).toEqual(1)
    })

    it(`renders create section by default`, () => {
        const wrapper = shallow(<NotFoundPage />)
        expect(toJson(wrapper)).toMatchSnapshot()
    });
})