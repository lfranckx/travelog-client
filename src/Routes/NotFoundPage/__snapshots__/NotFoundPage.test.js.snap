// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage component renders create section by default 1`] = `
<section
  className="NotFoundPage"
>
  <h2>
    404 - Page not found
  </h2>
  <p>
    Try going back to your previous page.
  </p>
</section>
`;
